Lib/importlib/machinery.py

```diff
@@ -14,6 +14,7 @@
 from ._bootstrap_external import ExtensionFileLoader
 from ._bootstrap_external import AppleFrameworkLoader
 from ._bootstrap_external import NamespaceLoader
+from ._bootstrap_external import NamespacePath
```

Comment:
Contrary to the title and what's new entry, this doesn't expose `importlib.NamespacePath`, but `importlib.machinery.NamespacePath`:

```python
>>> from importlib import NamespacePath
Traceback (most recent call last):
  File "<python-input-0>", line 1, in <module>
    from importlib import NamespacePath
ImportError: cannot import name 'NamespacePath' from 'importlib' (...)
>>> from importlib.machinery import NamespacePath
>>> 
```

Aside from what you intended, I'm not sure that this isn't actually better.  `importlib.NamespacePath` seems a little odd to me, and while something like `importlib.abc` *might* be better, but this isn't really an ABC.  So `importlib.machinery.NamespacePath` seems fine.  Care to chime in, @brettcannon ?

Reply:
Yes, sorry! I am okay with either namespace. From the lack of feedback, I think it should be okay to proceed with `importlib.machinery.NamespacePath`.